packages/core/src/utils/army.ts

```
@@ -1,312 +1,231 @@
-	}
+export const getTroopResourceId = (troopType: TroopType, troopTier: TroopTier): ResourcesIds => {
+  switch (troopType) {
+    case TroopType.Knight:
+      switch (troopTier) {
+        case TroopTier.T1:
+          return ResourcesIds.Knight;
+        case TroopTier.T2:
+          return ResourcesIds.KnightT2;
+        case TroopTier.T3:
+          return ResourcesIds.KnightT3;
+      }
```

Biome (1.9.4)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)